src/player.h

```diff
@@ -0,0 +1,128 @@
+/**
+ * @file player.h
+ * @brief Define a player for Numbers Royale
+ * Each player has a moves set and a score.
+ * Once a number is played, it is deleted from the moves set.
+ */
+#pragma once
+#include <unordered_set>
+#include <string>
+
+
+class Player {
+protected:
+  std::unordered_set<unsigned int> moves_;
+  unsigned int id_;
+  unsigned int score_;
+  unsigned int last_move_;
+  unsigned int board_size_;
```

Comment:
Maybe I'm too embedded now that I **never** use `int` and strictly will use `(s)size_t` and `(u)intX_t`

Reply:
I'll choose to let it be :)

Reply:
I think all the projects that I've worked on don't use int either. Always #define their own types for things, e.g. `#define CK_INT int`.

What's the (s) and (u)? signed unsigned long?

Reply:
`ssize_t` is a signed `size_t`
`uint8_t` is an unsigned `int8_t`

Reply:
C++:
```cpp
using CK_INT = int;
```
C:
```c
typedef CK_INT int;
```